cfn_encrypt/ssm_parameter.py

```diff
@@ -14,37 +17,41 @@ def parameter_exist(name):
     )
     return len(response["Parameters"]) > 0
 
+def get_propertry(rp, name:str=''):
```

Comment:
Is it allowed to have a dict with an empty string as a key? Would think the `name` param is mandatory

Reply:
Hmm is indeed more accurate to remove the empty key part